common/+MeasFilters/SingleShot.m

```diff
@@ -138,10 +147,10 @@
                     gPDF = ksdensity(intGroundIData(intPt,:), bins);
                     ePDF = ksdensity(intExcitedIData(intPt,:), bins);
                 else
-                    groundIData = sum(real(weightedGround)) - real(bias);
-                    excitedIData = sum(real(weightedExited)) - real(bias);
-                    groundQData = sum(imag(weightedGround)) - imag(bias);
-                    excitedQData = sum(imag(weightedExited)) - imag(bias);
+                    groundIData = sum(real(weightedGround)); %- real(bias);
```

Comment:
ops, I didn't mean to commit that change. But to get a 1:1 match between histograms (by matlab) and integrated voltages (by the card), we should not subtract a bias.